services/scan/src/store.ts

```diff
@@ -404,90 +516,142 @@ export class Store {
+        batches.deleted_at is null
+    `) as { ballotsCounted: number } | undefined;
+
+    return r?.ballotsCounted ?? 0;
   }
 
   /**
    * Returns whether the appropriate backups have been completed and it is safe
-   * to unconfigure a machine/zero out data. Always returns true in test mode.
+   * to unconfigure a machine / reset the election session. Always returns
+   * true in test mode.
    */
   getCanUnconfigure(): boolean {
-    // Always allow unconfiguring a machine in test mode
+    // Always allow in test mode
     if (this.getTestMode()) {
       return true;
     }
 
-    const scannerBackedUpAt = this.getBackupTimestamp(BackupKey.Scanner);
-
-    if (!this.batchStatus().length) {
+    // Allow if no ballots have been counted
+    if (!this.getBallotsCounted()) {
       return true;
     }
 
+    const scannerBackedUpAt = this.getScannerBackupTimestamp();
+
     // Require that a scanner backup has taken place
     if (!scannerBackedUpAt) {
       return false;
     }
 
-    const { maxCvrsCreatedAt, maxCvrsDeletedAt } = this.client.one(
-      'select max(created_at) as maxCvrsCreatedAt, max(deleted_at) as maxCvrsDeletedAt from sheets'
-    ) as {
-      maxCvrsCreatedAt: Iso8601Timestamp;
-      maxCvrsDeletedAt: Iso8601Timestamp;
+    // Adding or deleting sheets would have updated the CVR count
+    const { maxSheetsCreatedAt, maxSheetsDeletedAt } = this.client.one(`
+        select
+          max(created_at) as maxSheetsCreatedAt, 
+          max(deleted_at) as maxSheetsDeletedAt
+        from sheets
+      `) as {
+      maxSheetsCreatedAt: Iso8601Timestamp;
+      maxSheetsDeletedAt: Iso8601Timestamp;
     };
-    const { maxBatchesStartedAt, maxBatchesDeletedAt } = this.client.one(
-      'select max(started_at) as maxBatchesStartedAt, max(deleted_at) as maxBatchesDeletedAt from batches'
-    ) as {
-      maxBatchesStartedAt: Iso8601Timestamp;
+
+    // Deleting non-empty batches would have updated the CVR count
+    const { maxBatchesDeletedAt } = this.client.one(`
+      select
+        max(batches.deleted_at) as maxBatchesDeletedAt
+      from batches inner join sheets
+      on sheets.batch_id = batches.id
+      where sheets.deleted_at is null
+    `) as {
       maxBatchesDeletedAt: Iso8601Timestamp;
     };
 
-    const cvrsLastUpdatedAt = [maxCvrsCreatedAt, maxCvrsDeletedAt]
+    const cvrsLastUpdatedAt = [
+      maxBatchesDeletedAt,
+      maxSheetsCreatedAt,
+      maxSheetsDeletedAt,
+    ]
       .filter(Boolean)
       .reduce((max, curr) => (max > curr ? max : curr), '');
-    const batchesLastUpdatedAt = [maxBatchesStartedAt, maxBatchesDeletedAt]
-      .filter(Boolean)
-      .reduce((max, curr) => (max > curr ? max : curr), '');
-
-    if (!batchesLastUpdatedAt) {
-      return true;
-    }
 
-    const isBackupUpToDate =
-      scannerBackedUpAt >= cvrsLastUpdatedAt &&
-      scannerBackedUpAt >= batchesLastUpdatedAt;
-    return isBackupUpToDate;
+    return scannerBackedUpAt >= cvrsLastUpdatedAt;
```

Comment:
I realize all this was here before, but it makes me anxious that all this date math is being done with `string`s. Can we use `DateTime` from `luxon` or `Date` instead? Using strings assumes that all the values are in the timezones. That should be true here, but it feels like an unnecessarily fragile assumption. If not in this PR, can you make a follow-up ticket?

Reply:
I actually hit this issue when I tried to use something other than `current_timestamp` during testing. I think it makes sense, and I will include it in this PR, but will have you take a quick look.

Reply:
I've started using `DateTime`, but did you know what we're storing our dates (from `current_timestamp`) in non-timezoned GMT strings that don't even necessarily fit the `ISO8601` spec? E.g. `2022-11-16 18:18:54`. We should probably be storing with timezones, right?

Reply:
Oh no, really? grrr 😠 sqlite

Reply:
@kofi-q just changed it in the admin service: https://github.com/votingworks/vxsuite/pull/2789#discussion_r1025510235